Lib/typing.py

```diff
@@ -1670,21 +1669,6 @@ def __ror__(self, left):
         return Union[left, self]
 
 
-class _DeprecatedGenericAlias(_SpecialGenericAlias, _root=True):
```

Comment:
We might need this again in the future. But I guess it's pretty easy to add it back if/when that happens